manifests/devopsconsole/0.1.0/devopsconsole-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,124 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    capabilities: Full Lifecycle
+    description: Enables the DevOps perspective in OpenShift 4.x
+    categories: "Developer Tools"
+  name: devopsconsole-operator.v0.1.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: Component
+      name: components.devopsconsole.openshift.io
```

Comment:
Did you try installing this? I remember I had had trouble using this because of missing `displayName` and `description`

Reply:
Done. 236f76ad7b54564d126429f6c567deba729b12f1